tests/chart/test_dag_processor.py

```diff
@@ -0,0 +1,72 @@
+from tests.chart.helm_template_generator import render_chart
+
+from . import get_containers_by_name
+
+
+class TestDagProcessor:
```

Comment:
nit: Using a class isn't really helpful unless we're parametrizing the class. IMHO we shouldn't use pytest classes without parametrization.